statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -60,14 +64,16 @@ func (wr *WidgetsRender) setGrid() {
 	gridLeft := ui.NewGrid()
 
 	gridLeft.Set(
-		ui.NewRow(10.0/22, wr.instanceInfo),
-		ui.NewRow(12.0/22, wr.chainInfo))
+		ui.NewRow(5.0/11, wr.instanceInfo),
+		ui.NewRow(6.0/11, wr.chainInfo))
 
 	gridRight := ui.NewGrid()
-	gridRight.Set(ui.NewRow(1.0/4, wr.cpuLoad),
-		ui.NewRow(1.0/4, wr.memoryLoad),
-		ui.NewRow(1.0/4, wr.networkRecv),
-		ui.NewRow(1.0/4, wr.networkSent))
+	gridRight.Set(
+		ui.NewRow(5.0/11, wr.blockInfo),
+		ui.NewRow(1.5/11, wr.cpuLoad),
```

Comment:
Same as above, if you divide by 22, the newrow will have 3/22 (3 the actual height)

Reply:
Changed with 22 because is more clear.